src/main/resources/templates/pricing.html

```diff
@@ -15,20 +15,41 @@
 		<div class="row">
 			<div class="heading text-center col-sm-9 col-sm-offset-1">
 				<p>&nbsp;</p>
-				<p>&nbsp;</p>
-				<p>&nbsp;</p>
 				<div class="h15">
 					<p>Pricing</p>
 				</div>
 				<p>&nbsp;</p>
-				<h4>Optimize your spend for both variable or stable workloads </h4>
-			</div>
-			<div class="col-sm-10 col-sm-offset-1">
-				<p>More information will be published soon.</p>
+				<h4>As a not-for-profit organization, NCL offers competitive rate to all users.</h4>
+				<p>&nbsp;</p>
+				<div class='wrapper' th:align="center">
+					<div class='package brilliant'>
+							<div class='name' align="center">Promotion(50%)</div>
+							<p><s>SGD 0.24 per hour</s></p>
+							<h16>SGD 0.12 per hour</h16>
+						<p>&nbsp;</p>
+							<div align="center"><strong>From NOV 2016 - MAY 2017</strong></div>
+						<hr/>
+						<p align="left">- 24 Nodes available</p>
+						<p align="left">- Free access to all tools </p>
+						<p align="left">- Free access to available datasets  </p>
+						<p align="left">- Free access to services</p>
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this deprecated "align" attribute. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=Web%3AS1827)